root/etc/nethserver/todos.d/20ad_credentials

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+
+#
+# Copyright (C) 2015 Nethesis S.r.l.
+# http://www.nethesis.it - support@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+import gettext
+import json
+import subprocess
+import sys
+
+out = ''
+
+try:
+    out = json.load(subprocess.Popen(['/sbin/e-smith/config', 'printjson', 'sssd'], stdout=subprocess.PIPE).stdout)
```

Comment:
Disable this todo if ldap credentials are not required.

Reply:
Fixed by commit "Refactor ldap-credentials-optional UI helper"